### ISSUE TITLE
`model_profile` provided with extra argument in __main__.py

### ISSUE THREAD

**Issue by haojin2** [ThanatosShinji/onnx-tool#33]
**Describe the bug**
https://github.com/ThanatosShinji/onnx-tool/blob/main/onnx_tool/__main__.py#L93

**To Reproduce**
Just run a model profile with the __main__.py like: `python3 -m onnx_tool -m profile -i model.onnx`

**Comment by ThanatosShinji** [comment]
Do you mean need more args for model profile using from console?

**Comment by haojin2** [comment]
No, you need to remove `, dump_outputs=args.names` to make it work, otherwise you're providing an extra argument into that function call
